packages/web-react/src/components/Tooltip/__tests__/Tooltip.test.tsx

```diff
@@ -42,7 +42,7 @@ describe('Tooltip', () => {
         <TooltipPopover>{popoverText}</TooltipPopover>
       </Tooltip>,
     );
-    const element = dom.container.querySelector('.Tooltip') as HTMLElement;
+    const element = dom.container.querySelector('.TooltipPopover') as HTMLElement;
```

Comment:
Nice-to-have: refactor this test according to the new approach. :-)

Reply:
refactored 👍